src/emulator/types.ts

```diff
@@ -28,7 +28,7 @@ export function isJavaEmulator(value: string): value is JavaEmulators {
 }
 
 export function isEmulator(value: string): value is Emulators {
-  return Object.values(Emulators).indexOf(value) >= 0;
+  return Object.values(Emulators).indexOf(value as Emulators) >= 0;
```

Comment:
This one was weird and seems wrong but it's the only way I could get it to compile. TS seems like it should be smarter here.